cs_tests/cs15.c

```diff
@@ -1,9 +1,12 @@
 #include "aliascheck.h"
 void foo(int **w,int**x,int **y, int *z){
 	int *t;
-	t = *x;
 	*y = z;
-	*t = w;
+	t = *x;
+	*w = t;
+	MUSTALIAS(a1,&d1);
```

Comment:
where are a1 and d1? They are not defined in function foo. How can you compile this program?

Reply:
Were you trying to put them in `main`?

Reply:
Yes, I added them in the wrong function. In the latest commit I have added them in main.

Sorry, I made these mistakes as I was testing the test cases on my implementation, which is intra-procedural(had manually inlined the function calls).

Reply:
I still can not see why a1 points to d1 in your revised case.